systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -591,4 +591,9 @@ public static void waitUntilKafkaStatusContainsKafkaMetadataState(String namespa
             return k.getStatus().getKafkaMetadataState().equals(desiredKafkaMetadataState);
         });
     }
+
+    public static String getKafkaLogFolderNameInPod(String namespaceName, String kafkaPodName) {
+        return ResourceManager.cmdKubeClient().namespace(namespaceName)
+            .execInPod(kafkaPodName, "/bin/bash", "-c", "ls /var/lib/kafka/data | grep \"kafka-log[0-9]\" -o").out().trim();
```

Comment:
`kafka-log[0-9]` that suffix number is based on pod-number I think so if you deploy Kafka with ten replicas it will not match in such an edge case. Either make it to `[0-9]+` or add `kafkaPodNum` as a suffix?

Reply:
Yeah I was thinking that we will actually not have more than 10 replicas. Also the listing the dir instead of parsing the number of Pod from the name looked easier to me